src/backend/optimizer/plan/createplan.c

```diff
@@ -1319,7 +1317,7 @@ create_projection_plan(PlannerInfo *root, ProjectionPath *best_path)
 		copy_path_costsize(root, plan, (Path *) best_path);
 		plan->flow = cdbpathtoplan_create_flow(root,
 											   best_path->path.locus,
-											   best_path->path.parent ? best_path->path.parent->relids : NULL,
+											   best_path->path.parent->relids,
```

Comment:
Here `best_path->path.parent` cannot be NULL because it is dereferenced above in the function `tlist = build_path_tlist(root, &best_path->path)`.